flink-table/flink-table-planner/src/main/java/org/apache/flink/table/sqlexec/SqlToOperationConverter.java

```diff
@@ -193,6 +195,8 @@ private SqlToOperationConverter(
 			return Optional.of(converter.convertShowCurrentDatabase((SqlShowCurrentDatabase) validated));
 		} else if (validated instanceof SqlShowTables) {
 			return Optional.of(converter.convertShowTables((SqlShowTables) validated));
+		} else if (validated instanceof SqlShowCreateTable) {
```

Comment:
I think we don't need to support this feature on the old planner right?

Reply:
Yes. I agree with you.

Reply:
Okay. Then I'll revert this change.